delta/src/main/scala/ch/epfl/bluebrain/nexus/delta/RepairFromMessagesClassic.scala

```diff
@@ -0,0 +1,117 @@
+package ch.epfl.bluebrain.nexus.delta
+
+import java.net.URLDecoder
+import java.util.UUID
+
+import akka.actor.ActorSystem
+import akka.persistence.cassandra.query.scaladsl.CassandraReadJournal
+import akka.persistence.query.PersistenceQuery
+import ch.epfl.bluebrain.nexus.admin.organizations.Organizations
+import ch.epfl.bluebrain.nexus.admin.projects.Projects
+import ch.epfl.bluebrain.nexus.delta.config.AppConfig
+import ch.epfl.bluebrain.nexus.iam.acls.Acls
+import ch.epfl.bluebrain.nexus.iam.permissions.Permissions
+import ch.epfl.bluebrain.nexus.iam.realms.Realms
+import ch.epfl.bluebrain.nexus.kg.resources.ProjectIdentifier.ProjectRef
+import ch.epfl.bluebrain.nexus.kg.resources.{Id, Repo, ResId}
+import ch.epfl.bluebrain.nexus.rdf.Iri
+import com.typesafe.scalalogging.Logger
+import monix.eval.Task
+import monix.execution.Scheduler
+import monix.execution.schedulers.CanBlock
+import retry.CatsEffect._
+import retry.RetryPolicies.{exponentialBackoff, limitRetries}
+import retry.implicits.retrySyntaxBase
+import retry.{RetryDetails, RetryPolicy}
+
+import scala.concurrent.duration.DurationInt
+import scala.util.Try
+import scala.util.control.NonFatal
+
+/**
+  * Repair tool to rebuild the dependent tables in case tag_views is out of sync with the messages tables.
+  */
+object RepairFromMessagesClassic {
+  // $COVERAGE-OFF$
+
+  private val log = Logger[RepairFromMessagesClassic.type]
+
+  private val parallelism = sys.env.getOrElse("REPAIR_FROM_MESSAGES_PARALLELISM", "1").toIntOption.getOrElse(1)
```

Comment:
You could have use a property from `migration.conf` (MigrationConfig in AppConfig), since this already exists and it is used for this purpose.